canonical-websites/www.ubuntu.com
# Update linuxone thankyou page

## Done

- Update page copy
- Rename page/URL
- Add redirect

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/download/server/thank-you-s390x>
- Check that page matches [the copy doc](https://docs.google.com/document/d/1GptXtMfbgOjOmqb8DZCCTRwogkiP_cM95U9Z0gFSt7g/edit)
- View the site locally in your web browser at: <http://0.0.0.0:8001/download/server/thank-you-linuxone> and check that you are redirected to <http://0.0.0.0:8001/download/server/thank-you-s390x>


## Issue / Card

Fixes #2962
